WORKSPACE

```diff
@@ -1,5 +1,11 @@
 workspace(name = "org_tensorflow")
 
+load("//tensorflow:workspace.bzl", "check_version", "tf_workspace")
+
+# DO NOT ADD ANYTHING ABOVE THIS LINE!!!
```

Comment:
Can we be a bit more careful about what is said here?  Maybe we can say "We must check the bazel version before trying to parse any other BUILD files, in case the parsing of those build files depends on the bazel version we require here."